### DISCUSSION TITLE
Can we see user "Online" or "Last Seen" like other messengers?

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#142] [Q&A/Support] [answered]
I know snapchat doens't have any option to see whether an user is online or not. And I'm not sure if you can get that using mod. That's what I want to ask that is it possible?

I guess you would have already tried that it the past.

**Comment by Pandasapling** [comment]
The only way you could see when they’re online is if they have their snapmap location on

**Reply by RevealedSoulEven** [reply]
Aaahhhhh
 I was asking if it's possible to add in snapmod to know whether someone is online. If snapchat provides any hidden API or something

**Comment by rodit** [comment, marked as answer]
From my knowledge, there is no such feature of the API unfortunately. I can have a look but one thing Snapchat is quite good at these days is not leaking more information than required (with the exception of birthdays haha, although these are required for the astrology thing in the friend profile).
